exalearn-ui/package-lock.json

```diff
@@ -145,65 +145,6 @@
         "rxjs": "6.6.7"
       }
     },
-    "@angular-eslint/builder": {
```

Comment:
Are you sure you want to remove all these dependencies? Did you discuss it with @HrebenYury?

Reply:
For some reason it was made automatically, now returned